src/library/Uno.Toolkit.WinUI.Markup/ResourceValue.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Uno.Extensions.Markup;
+
+namespace Uno.Toolkit.Markup
+{
+	public record struct ResourceValue<T>
```

Comment:
since this is immutable, we should add the `readonly` modifier.
```suggestion
	public readonly record struct ResourceValue<T>
```